tools/metrics/datadog_reporter.go

```diff
@@ -103,7 +103,7 @@ func (r *DatadogReporter) WriteMetrics(events []Metric) error {
 			continue
 		}
 		m := datadog.Metric{}
-		m.SetMetric("op." + ev.Name)
```

Comment:
I can't see anything in the commit messages that explains why we've put a bunch of metrics under a new namespace(?). I feel like this might be something useful to have a record of.

Reply:
This was a refactor - the `op.` namespace used to be added centrally, but we wanted a new namespace, so un-DRYed it first. This is what we meant by "metric name prefixes are now part of names" in https://github.com/alphagov/paas-cf/pull/1127/commits/26d221e626d0a066410943dc8394dc878e8f6823